src/widget_macro.md

```diff
@@ -2,6 +2,10 @@
 
 To simplify the implementation of the `Widgets` trait, let's use the relm4-macros crate!
 
+![App screenshot dark](img/screenshots/simple-dark.png)
+
+The app will look and behave identical to one implemented in the previous chapter. Only the way it is implemented will differ.
```

Comment:
The app will look and behave identical to our first app from the previous chapter. Only the implementation is different.